modules/statistics/php/stats_demographic.class.inc

```diff
@@ -143,15 +144,14 @@ class Stats_Demographic extends \NDB_Form
     function setup()
     {
         parent::setup();
-
-        $DB          =& \Database::singleton();
+        $db          =& \Database::singleton();
```

Comment:
If you're changing this line I think you may as well remove the ampersand `=&` -> `=`